eng/versioning/version_client.txt

```diff
@@ -83,6 +83,7 @@ com.azure.resourcemanager:azure-resourcemanager-privatedns;2.0.0-beta.4;2.0.0-be
 com.azure.resourcemanager:azure-resourcemanager-resources;2.0.0-beta.3;2.0.0-beta.4
 com.azure.resourcemanager:azure-resourcemanager-redis;2.0.0-beta.3;2.0.0-beta.4
 com.azure.resourcemanager:azure-resourcemanager-samples;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-servicebus;2.0.0-beta.4;2.0.0-beta.4
```

Comment:
This is starting at `2.0.0-beta.4` because the package under the `com.microsoft` group left off at `2.0.0-beta.3`?

Reply:
It's because we have a centralized package [azure-resourcemanager](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/resourcemanager/azure-resourcemanager/pom.xml#L18). All the supported services named as `azure-resourcemanager-{serviceName}` would have same version. So for newly added one, it will have the same version defined in current version. 
Meanwhile, in `azure-resourcemanager` we just wrap all services into azure client so that the users could access them more conveniently.